docs/receipt-verifier.md

```diff
@@ -0,0 +1,118 @@
+---
+id: receipt-verifier
+title: Receipt Verifier Service
+sidebar_label: Receipt verifier service
+---
+
+This page explains how to set up a receipt verifier service.
+
+As more creators adopt Web Monetization to serve premium and/or ad-free content, there’s a greater need for reliable payment verification. Web Monetization browser events, like `monetizationstart`, are helpful indicators of payment, but can be spoofed by savvy users to access exclusive content without paying.
+
+[Interledger STREAM receipts](https://interledger.org/rfcs/0039-stream-receipts/) provide payment recipients (such as web monetized site owners) with verifiable proofs of payment. As a payment recipient, you can verify STREAM receipts yourself by setting up your own verification service or by using a third-party service. You don't need to run your own Interledger connector or SPSP server to verify STREAM receipts.
+
+## Before you begin
+
+Your Web Monetization receiver (the digital entity receiving payments) must be set up to generate Interledger STREAM receipts. No action is required on your part. GateHub and Uphold already support receipts.
+
+## Service overview
+
+The receipt verifier service acts as a proxy between the Web Monetization sender and receiver.
+
+Before a payment is made, the WM sender sends a query to the WM receiver. The query passes through the verifier service, which adds a receipt secret and receipt nonce.
+
+When the WM receiver receives money from the payment, it generates a receipt (which includes a signature generated using the receipt secret) and sends the receipt to the WM sender.
+
+The WM sender passes the receipt to the verifier service so the verifier can confirm the payment (as only the WM receiver and verifier service know the receipt secret). The verifier must verify the receipt before accepting the amount as paid.
```

Comment:
this portion is implemented by the site, but this paragraph makes it sound a bit like it happens automatically. The site will have to take the receipt from the web monetization events and then submit it to its backend (where the backend can submit the receipt to the receipt verifier service).

Reply:
@sharafian How about this:

The site takes the receipt from the Web Monetization events, then submits the receipt to its backend. The backend then submits the receipt to the receipt verifier service so the service can confirm the payment (as only the WM receiver and the receipt verifier service know the receipt secret). The service must verify the receipt before accepting the amount as paid.